app/assets/stylesheets/guests.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the guests controller here.
```

Comment:
clear unused generated files

Reply:
Won't we use this file later when we start adding styles?